docs/Deploying/Debugging.md

```diff
@@ -0,0 +1,156 @@
+# Debugging
+
+## Goals of this guide
+
+This is NOT meant to be an exhaustive list of possible issues in Fleet and how to solve them.
+
+This is a guide for going from a vague statement such as "things are not working correctly" to a more narrowed down and 
+specific assessment. This doesn't mean necessarily a solution; but, with a more specific assessment, it'll be easier for 
+the Engineering team to help.
+
+Note that even if you do all your homework, the Engineering team might have follow-up questions.
+
+## Basic data that is needed
+
+While it's not needed strictly 100% of the times, in most cases it's extremely useful to have a clear understanding of 
+the basic characteristics of the Fleet deployment with the issues:
+
+- Amount of total hosts.
+- Amount of online hosts.
+- Amount and size (CPU/Mem) of the Fleet instances.
+- Fleet instances CPU and Memory usage while the issue has been happening.
+- MySQL flavor/version in use.
+- MySQL database size (CPU/Mem).
+- MySQL CPU and Memory usage while the issue has been happening.
+- Are database readers configured? If so, how many?
```

Comment:
Is this another name for "read replicas"?

Reply:
Yeah, I should just use read replicas.